### PR TITLE
feature: add a left menu option to add a below block

### PR DESCRIPTION
**Pull request by chenmoonmo**
When i create a code or media block, cant add below block easily.

The PR adds a left menu option to add a below block.

### CODE REVIEW

File: libs/components/editor-plugins/src/menu/left-menu/LeftMenu.tsx [hunk 1]
```diff
@@ -45,6 +46,22 @@ export function LeftMenu(props: LeftMenuProps) {
                 ),
                 icon: <TurnIntoIcon />,
             },
+            {
+                title: 'Add A Below Block  ',
```
Comment (darkskygit) [inline libs/components/editor-plugins/src/menu/left-menu/LeftMenu.tsx:50]: has over spaces

File: libs/components/editor-plugins/src/menu/left-menu/LeftMenu.tsx [hunk 2]
```diff
@@ -45,6 +46,22 @@ export function LeftMenu(props: LeftMenuProps) {
...
+                icon: <AddViewIcon />,
+                callback: async () => {
+                    const block = await editor.getBlockById(blockId);
+                    const belowType = ['numbered', 'todo', 'bullet'].includes(
```
Comment (lawvs) [inline libs/components/editor-plugins/src/menu/left-menu/LeftMenu.tsx:54]: Please use the predefined types.

```ts
import { Protocol } from '@toeverything/datasource/db-service';

const belowType = [Protocol.Block.Type.bullet,Protocol.Block.Type.numbered, Protocol.Block.Type.todo]
```

File: libs/components/editor-plugins/src/menu/left-menu/LeftMenu.tsx [hunk 3]
```diff
@@ -45,6 +46,22 @@ export function LeftMenu(props: LeftMenuProps) {
...
+                        block.type
+                    )
+                        ? block.type
+                        : 'text';
```
Comment (lawvs) [inline libs/components/editor-plugins/src/menu/left-menu/LeftMenu.tsx:58]: Same here

```suggestion
                        : Protocol.Block.Type.text;
```

**Review by lawvs — CHANGES_REQUESTED** [review]
Thanks for your pull request.

**Review by lawvs — APPROVED** [review]
LGTM

**Pull request merged**